lib/commands/client-assertion.js

```diff
@@ -7,10 +7,10 @@ export default async function clientAssertion(clientId, privateKeyPath) {
   // Obtain an access token to call banno apis
   const enterpriseOidcTokenUri = 'https://banno.com/a/oidc-provider/api/v0/token';
 
-  clientAssertion = await signJWT(clientId, privateKeyPath)
+  const jwt = await signJWT(clientId, privateKeyPath)
```

Comment:
**thought:** Ah, this makes sense to me.

I had misinterpreted this as an okay thing to do since it _sort of_ looks like the 'shadow' concept for variables in Swift.

I probably should have asked about this, rather than 🤷 and moving on since it apparently worked. Today I learned.